src/models/workOrder.test.js

```diff
@@ -96,7 +96,20 @@ describe('WorkOrder', () => {
 
       MockDate.set(subMinutes(now, 1))
 
-      expect(workOrder.appointmentStartTimePassed()).toBe(false)
+      expect(workOrder.appointmentISODatePassed()).toBe(true)
+    })
+
+    it('returns true when the current time is before the appointment start time', () => {
```

Comment:
since we do, `subDays`
maybe it should be " it returns true when the current day is before the app day"?